firmware/controllers/actuators/idle_thread.cpp

```diff
@@ -110,10 +110,18 @@ percent_t IdleController::getRunningOpenLoop(float clt, SensorResult tps) {
 
 	running += iacByTpsTaper;
 
+	float airTaperRpmUpperLimit = engineConfiguration->idlePidRpmUpperLimit + engineConfiguration->airTaperRpmRange;
+	iacByRpmTaper = interpolateClamped(
```

Comment:
I don't understand this logic, why do we add more position at higher RPM?

This seems like it would encourage the engine to stall as it returns to idle.

![image](https://user-images.githubusercontent.com/568254/186786739-24dfc28d-4224-4f3f-8a04-2fcf2ace5f12.png)

Reply:
this is also not what #3940 asked for, that issue is asking for "predictive idle entry" where we look at current RPM, dRPM/dt, and some time window in the future about whether we're going to enter idle soon